test/built-ins/Temporal/PlainDateTime/prototype/since/units-changed.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.since
+description: Largest unit is respected
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const feb20 = new Temporal.PlainDateTime(2020, 2, 1, 0, 0);
+const feb21 = new Temporal.PlainDateTime(2021, 2, 1, 0, 0);
+
+TemporalHelpers.assertDuration(
+  feb21.since(feb20, { largestUnit: "years" }),
+  1, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+  "can return lower or higher units (years)"
```

Comment:
This assertion message and the ones below don't quite fit, to me. How about "returns no units higher than [unit]"?

Reply:
soudns good -- done